deepchecks/tabular/checks/model_evaluation/simple_model_comparison.py

```diff
@@ -257,21 +298,22 @@ def _create_simple_model(self, train_ds: Dataset, task_type: TaskType):
         Raises
         ------
         NotImplementedError
-            If the simple_model_type is not supported
+            If the strategy is not supported
         """
         np.random.seed(self.random_state)
 
-        if self.simple_model_type == 'random':
+        if self.strategy == 'uniform':
+            simple_model = UniformModel()
```

Comment:
Suggestion: If regression can randomly select from [min,max] instead of available labels